### DISCUSSION TITLE
Descripcion more details

### DISCUSSION THREAD

**Discussion by stecaptor** [rfennell/AzurePipelines#1094] [Ideas]
When displaying the description of a PR, it shows a text of maximum 420 characters.
I do not know if that truncation could be extended or some function that allows to display text
Thank you

**Comment by rfennell** [comment]
All the task does is return the JSON payload returned by the Azure DevOps API. The truncation you report seems to be a known issue as reported one  [developercommunity.visualstudio.com](https://developercommunity.visualstudio.com/t/rest-api-for-pull-requests-is-truncating-pr-descri/1543906). Helpfully they will fix it at some point.

**Reply by stecaptor** [reply]
But in the rest of the azure api the description of the PR if it is shown complete

Example
"title": "Habilitación de los estados de mantenimiento",
"description": "Se actualiza la versión del proyecto Bccr.WebApi.HealthChecks para poder habilitar los estados de mantenimiento de las Api por medio de nuevos llamados.\n\nCambios:\n1.\tBccr.WebApi.HealthChecks version 3.1.1.1-Preview1.2\n2.\tSe agregan la secciones: \n2.1\tservices.AddSingleton(new ApiStatus());\n2.2\t.AddCheck("Api", tags: new[] { "Live" })\n\nPruebas:\n\nLa colección de Postman en la ruta Bccr.OAuth2.DatosDelContexto.Api>Pruebas>Postam; fue actualizada con los nuevos llamados para sus respectivas pruebas:\n1. CurrentStatus\n2. MaintenanceRequested\n3. MaintenanceRunning\n4. Online\n5. Live\n\n",
"sourceRefName": "refs/heads/ApiMantenimiento",

But in the relase notes text task it only shows the description up to 420 characters

**Reply by rfennell** [reply]
I think it is an issue of the Azure DevOps Node SDK, which wrappers the API
